From: ROCKNIXK developers
Subject: [PATCH] sun4i-h616: retain H700 playback defaults

The H616 configuration is now provided upstream. Set the H700 playback
defaults in those files without replacing the upstream conf.d symlink.

--- a/ucm2/Allwinner/sun4i-h616/HiFi.conf
+++ b/ucm2/Allwinner/sun4i-h616/HiFi.conf
@@ -2,6 +2,7 @@
 	Comment "Internal Speaker"
 
 	EnableSequence [
+		cset "name='DAC Playback Volume' 63"
 		cset "name='Speaker Switch' on"
 	]
 
@@ -26,6 +27,10 @@
 SectionDevice."Headphones" {
 	Comment "Headphones"
 
+	EnableSequence [
+		cset "name='DAC Playback Volume' 63"
+	]
+
 	Value {
 		PlaybackMixerElem "Line Out"
 		PlaybackPCM "hw:${CardId},0"
--- a/ucm2/Allwinner/sun4i-h616/h616-audio-codec.conf
+++ b/ucm2/Allwinner/sun4i-h616/h616-audio-codec.conf
@@ -7,3 +7,7 @@
 	File "/Allwinner/sun4i-h616/HiFi.conf"
 	Comment "Play HiFi quality Music"
 }
+
+FixedBootSequence [
+	cset "name='DAC Playback Switch' on"
+]
